stage1/init/common/pod.go

```diff
@@ -406,12 +406,8 @@ func generateDeviceAllows(root string, appName types.ACName, mountPoints []types
 			link := filepath.Join(rktVolumeLinksPath, v.Name.String())
 
 			err := os.Symlink(tgt, link)
-			switch {
-			case err == nil:
-				toShift = append(toShift, tgt)
-			case os.IsExist(err):
-				// it already exists, do nothing
-			default:
+			// if the link already exists, we don't need to do anything
+			if err != nil && !os.IsExist(err) {
```

Comment:
Is `os.IsExist()` possible? `os.Symlink()` only returns `LinkError`:

https://golang.org/src/os/file_unix.go?s=10131:10174#L360